src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVM.md

```diff
@@ -337,7 +352,22 @@ The name of a new (or existing) network security group (NSG) for the created VM
 
 ```yaml
 Type: String
-Parameter Sets: StrategyParameterSet
+Parameter Sets: SimpleParameterSet
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Size
```

Comment:
@sergey-shandar Looks like you're missing a description in the New-AzureRMVM cmdlet.  Can you update the .cs file and regenerate the help file?

Discussed offline and Sergey will fix.

Reply:
Looks good.